### PR TITLE
fix the checkNotNull `!!` missing 

### LINKED ISSUE #135: check-not-null operator `!!` missing when printing
check-not-null operator `!!` missing when printing

Example case:
```kotlin
fun foo() {
    val l = listOf ( "x" )
    val a = l [ 0 ] !!
}
```

### PR DESCRIPTION
**Pull request by kunli2**
Fixes #135 
1. Fix the checkNotNull `!!` missing 
2. Add a failing test of `extensionFunction`

### CODE REVIEW

File: src/test/java/org/openrewrite/kotlin/tree/ExtensionFunctionTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+class ExtensionFunctionTest implements RewriteTest {
+
+    @Disabled
+    @Test
+    void extensionFunction() {
+        rewriteRun(
+          kotlin(
+            """
+              fun String.escape() = this.replace("a", "b")"
```
Comment (traceyyoshima) [inline src/test/java/org/openrewrite/kotlin/tree/ExtensionFunctionTest.java:32]:
```suggestion
              fun String . escape ( ) = this . replace ( "a" , "b" )"
```

Comment (traceyyoshima) [reply src/test/java/org/openrewrite/kotlin/tree/ExtensionFunctionTest.java:32]: We typically add spaces where possible in the `tree` tests to assert the whitespace is preserved

File: src/test/java/org/openrewrite/kotlin/tree/VariableDeclarationTest.java [hunk 2]
```diff
@@ -370,4 +370,17 @@ class Test {
...
         );
     }
+
+    @Test
+    void nonNull() {
+        rewriteRun(
+          kotlin("""
+            fun foo() {
+                val l = listOf("x")
+                val a = l[0] !!
```
Comment (traceyyoshima) [inline src/test/java/org/openrewrite/kotlin/tree/VariableDeclarationTest.java:380]:
```suggestion
                val a = l [ 0 ] !!
```

File: src/test/java/org/openrewrite/kotlin/tree/ExtensionFunctionTest.java (hunk 1 above)
Comment (kunli2) [reply src/test/java/org/openrewrite/kotlin/tree/ExtensionFunctionTest.java]: I see.

**Review by traceyyoshima — APPROVED** [review]
